CONTRIBUTING.md

```diff
@@ -9,21 +9,20 @@ The team welcomes contributions!  To make code changes to one of the Conjure rep
 ## Local Development
 
 ### Prerequisites
-- Python2 (On macOS: `brew install python2`)
 - Python3 (On macOS: `brew install python3`)
 - [pipenv](https://github.com/pypa/pipenv) (`pip3 install pipenv`)
 
-_We recommend the free [VSCode](https://code.visualstudio.com/) editor to work on python projects._
+_We recommend [VSCode](https://code.visualstudio.com/) to work on python projects._
 
 ### One-time setup for development
 
 1. Fork the repo
-1. Create the virtual environment `PIPENV_VENV_IN_PROJECT=1 pipenv --python 3 shell`
-1. Install all dependencies `pipenv install --dev`
-1. Download and generate integration test dependencies `./scripts/prepare_integration_tests.sh`
-1. From within a pipenv shell, run `code .` to open a new VSCode window.
+2. Create the virtual environment `PIPENV_VENV_IN_PROJECT=1 pipenv --python 3 shell`
```

Comment:
This isn't necessary since markdown will already show this as a sequential list, but I guess it doesn't hurt.